pyproject.toml

```diff
@@ -21,15 +21,16 @@ classifiers = [
 ]
 keywords = ["lsst"]
 dependencies = [
-    "astropy >= 4",
-    "sqlalchemy >= 1.4",
-    "click >= 7",
-    "pyyaml >= 6",
-    "pydantic >= 2, < 3",
+    "astropy >= 6.0.0",
+    "numpy < 2",
```

Comment:
You don't need this if numpy 2 works fine.

Reply:
Using numpy 2 results in an annoying error message printing at the top when the command line tools are run, which is why I added this. The stack is still using a numpy 1.x version as well. (numpy is not a direct dependency of felis though.)

Reply:
Stack using numpy 1.x is orthogonal to whether we allow other users to install this at all with numpy 2. What's the error message? Is it from astropy?

Reply:
No, it's an error message from numpy itself that I had trouble making go away.

Reply:
What is the error message

Reply:
I would also think that if we'll be doing additional felis development using astropy soon (e.g., generating astropy tables), the numpy version should match that from the stack.

Reply:
I disagree for the pypi version. It's the whole point of having tests on GitHub rather than locking ourselves to rubinenv (which can easily test for using Jenkins). What is the numpy2 error message?

Reply:
numpy error message:

https://github.com/lsst/felis/pull/109#discussion_r1829788091

Reply:
Thanks. `numpy.longdouble` does exist in numpy 2. It's a bit of an odd message. Really you want the stack trace from that warning.

Reply:
I'm going to remove the numpy requirement entirely, given that astropy has it as a transitive dependency and felis does not use it directly.